INSTALLING.md

```diff
@@ -2,199 +2,266 @@
 
 These instructions will guide you through the installation of Tenstorrent system tools and drivers, followed by the installation of TT-Metalium and TT-NN.
 
-> [!IMPORTANT]
->
-> If you are using a release version of this software, please check the installation instructions packaged with that version. You can find them in either the release assets for that version, or in the source files for that version tag.
-
 ---
 
-## Installation Steps
+## Prerequisites:
 
-### Step 1. Driver & Firmware
+### 1: [Setup the Hardware](https://docs.tenstorrent.com/quickstart.html#unboxing-and-hardware-setup)
 
-Follow the Software Setup instructions for your specific board or system provided on our [general docs](https://docs.tenstorrent.com).
+Once you have setup the hardware move on to step 2.
 
-If you have purchased a Grayskull card, you will find the instructions [here](https://docs.tenstorrent.com/aibs/grayskull/installation.html).
+---
 
-Note the current compatability matrix:
+### 2: Install Driver & Firmware
 
-| Device              | OS              | Python   | Driver (TT-KMD)    | Firmware (TT-Flash)                        | TT-SMI                | TT-Topology                    |
-|---------------------|-----------------|----------|--------------------|--------------------------------------------|-----------------------|--------------------------------|
-| Grayskull           | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.9.0.0 (v80.9.0.0)               | v2.2.0 or above       | N/A                            |
-| Wormhole            | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.10.0.0 (v80.10.0.0)             | v2.2.0 or above       | N/A                            |
-| T3000 (Wormhole)    | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.10.0.0 (v80.10.0.0)             | v2.2.0 or above       | v1.1.3 or above, `mesh` config |
+#### Install System-level Dependencies
```

Comment:
Just to be clear these are not required for Docker Release install

Reply:
I kept it there for now because many of these dependencies are required during the driver and firmware installation. I believe having it as a first step makes things flow easier. On the other hand we should definitely look for better alternatives to wrap things together. Let me know if keeping it there is not an option.